src/node_util.cc

```diff
@@ -174,6 +176,20 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Local<String> envtag = args[0]->ToString();
```

Comment:
Can you use the overload that takes a `Local<Context>`?

Or simply use `args[0].As<String>()` and `CHECK(args[0]->IsString())` first.  It's an internal API, it's reasonable to enforce strict types.

Reply:
Can you point me where can I find different overload options?

Reply:
Yep, search `deps/v8/include/v8.h` for ToString.

Reply:
thanks!

Reply:
@bnoordhuis as a side benefit, can you point me somewhere to read about v8(guides are out of date or poor), mostly to improve my contributions.

Reply:
There is some documentation on the V8 wiki and a few samples in the V8 repo but that's about it.  Working with V8 means reading its source (or at least its header files.)

At least the headers have pretty good doc comments now.  It wasn't anything like that when I started working on node.

Reply:
will have that in mind, thanks!